datamod/chapter_1/items/items/choc_diamond.lua

```diff
@@ -0,0 +1,9 @@
+return "choc_diamond", function(item)
+    item.heal_variants_battle = {
+        ["kris"] = 80, 
+        ["susie"] = 30, 
+        ["ralsei"] = 30, 
+        ["noelle"] = 50,
+        ["default"] = 30
```

Comment:
bc of other suggested changes, remove "default" here

```suggestion
        ["noelle"] = 50
```